Makefile

```diff
@@ -0,0 +1,23 @@
+BIND_DIR := build
+MKDIR_P = mkdir -p
+RM = rm -rf
+
+default: build-all
+
+build-jessie:
+	$(MAKE) clean-up
+	${MKDIR_P} ${CURDIR}/${BIND_DIR}
+	./pre-build.sh
```

Comment:
Running this on every build is going to be very annoying as it does a slow `apt-get update`.

I would suggest this is just an optional target that we can document to run when you first
clone the project to get the deps installed.

Otherwise it should perhaps use a target with a stamp file so that it doesn't run every time.